modules/nf-core/custom/shortenfastaids/templates/shortenfastaids.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
```

Comment:
Can we get some LICENSE and authoring info on this script?

Reply:
Thank you @maxulysse 

I have added the authoring and LICENSE info.